### PR TITLE
Version bump.

### PR DESCRIPTION
**Pull request by pq**
Bump to tag package discovery logic.

@sethladd @lrhn

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ environment:
 
 dependencies:
   charcode: '^1.1.0'
-  path: "any"
-  http: "any"
+  path: 'any'
```
Comment (sethladd) [inline pubspec.yaml]: let's set these to specific version ranges

**Comment by pq** [comment]
@hterkelsen

File: pubspec.yaml (hunk 1 above)
Comment (zoechi) [inline pubspec.yaml]: Shouldn't there be concrete ranges?

Comment (pq) [inline pubspec.yaml]: I'll take a stab.  According to Lasse, these are broad for expediency.

Comment (pq) [inline pubspec.yaml]: @sethladd done.  PTAL!

**Comment by sethladd** [comment]
lgtm thanks!

**Pull request merged**